### PR TITLE
Adding markdownlint on extension.json and GitHub Actions

### PR DESCRIPTION
**Pull request by Ikuyadeu**
**What problem did you solve?**

To avoid markdown-related issues (e.g. #587), we will use markdownlint.

TODO:
* [x] Add markdownlint on the package.json
* [x] Add markdownlint and get warnings
* [x] Fix current document for the markdownlint

~~p.s. I missed status to WIP, This PR cannot merge yet.~~

### CODE REVIEW

**Comment by Ikuyadeu** [comment]
Now, this is readied.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+# Latest updates
+
+From v1.6.6, You can check all of our changes from [Releae Page](https://github.com/Ikuyadeu/vscode-R/releases)
```
Comment (renkun-ken) [inline CHANGELOG.md:5]: typo `Releae`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,30 +25,18 @@ Thank you for contributors
...
-* Move `r.runSource` and `r.knitRmd` to `editor/title/run` #573 (Fix #572)
-* Fix so code can be run after creating terminal #567
-* Add option to keep terminal hidden after running code #566
-* Scroll to bottom after running a command #559 (Thank you @samkimhis)
-* Refactoring and implementation of webviewPanelSerializer #556
-* add option vsc.hover.str.max.level #545
-* Change workspace tooltip #544 (Thank you @ElianHugh)
-
-
 ## 1,6,4
```
Comment (renkun-ken) [inline CHANGELOG.md:28]: The title use `,` instead of `.`

File: CHANGELOG.md (hunk 1 above)
Comment (Ikuyadeu) [reply CHANGELOG.md]: Yes, I should add the typo checker in the future.

File: CHANGELOG.md (hunk 2 above)
Comment (Ikuyadeu) [reply CHANGELOG.md]: Thank you. I can not understand what did I think.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,19 +1,20 @@
 # Change Log
 
-# Latest updates
+## Latest updates
 
-From v1.6.6, You can check all of our changes from [Releae Page](https://github.com/Ikuyadeu/vscode-R/releases)
+From v1.6.6, You can check all of our changes from [Release Page](https://github.com/Ikuyadeu/vscode-R/releases)
 
-# [1.6.6](https://github.com/Ikuyadeu/vscode-R/releases/tag/v1.6.6)
+## [1.6.6](https://github.com/Ikuyadeu/vscode-R/releases/tag/v1.6.6)
```
Comment (renkun-ken) [inline CHANGELOG.md:7]: Does this mean we don't write updates from collaborators in this changelog any more? It looks a bit un-informative as a quick preview of what has been updated in this release. A typical practice is that the changelog lists the most important updates and the release page shows the updates in details.

Comment (Ikuyadeu) [reply CHANGELOG.md:7]: I see, actually, this is unhelpful.
So I'll add important update information as the `Highlight` it does not include the engineering update such as lint fix and refactoring like this PR.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,10 +2,17 @@
...
 ## [1.6.6](https://github.com/Ikuyadeu/vscode-R/releases/tag/v1.6.6)
 
+Highlight
+
+* Clarify error messages
+* Being more conservative to call object.size() in task callback
+* Send code to debug repl
+* shim the rstudioapi if it has already been loaded
+
 Thank you for contributors
```
Comment (renkun-ken) [inline CHANGELOG.md:16]: Maybe "Thank you for your contributions:" sound better?

File: CHANGELOG.md (hunk 3 above)
Comment (renkun-ken) [reply CHANGELOG.md:7]: Yes exactly, I guess most users only care about new features and some important bug fixes and changes of behavior.

Comment (renkun-ken) [reply CHANGELOG.md:7]: In the future, we may demand the author of a PR to update the changelog if we think the PR is important to mention and then we could review the changelog updates too within the PR.

Comment (Ikuyadeu) [reply CHANGELOG.md:7]: I agree with you.
Official vscode makes the PR changelog automatically. We may imitate that approach.

File: CHANGELOG.md (hunk 4 above)
Comment (Ikuyadeu) [reply CHANGELOG.md]: Yes, that is more natural.

**Pull request merged**